livre/eq/render/RayCastRenderer.cpp

```diff
@@ -430,9 +429,13 @@ struct RayCastRenderer::Impl
 
     void onFrameRender( const NodeIds& bricks )
     {
+        createAndFillVertexBuffer( bricks );
         size_t index = 0;
         for( const NodeId& brick: bricks )
             renderBrick( brick, index++ );
+
+        glDeleteBuffers( 1, &_posVBO );
+        glFlush();
```

Comment:
My point exactly. GL functions, malloc, fopen all have a symmetric destruction counterpart (free, close, etc.) and return the resource to be freed.

You are right, these are C-style APIs and ideally the setupBuffer function would return some smart_ptr which self-destructs, but this seems to be overkill here.